projects/testing-library/src/lib/testing-library.ts

```diff
@@ -386,7 +386,9 @@ function cleanupAtFixture(fixture: ComponentFixture<any>) {
 // this ensures that tests run in isolation from each other
 // if you don't like this, set the ATL_SKIP_AUTO_CLEANUP env variable to 'true'
 if (typeof process === 'undefined' || !process.env?.ATL_SKIP_AUTO_CLEANUP) {
```

Comment:
Is this still needed?
The auto-cleanup?

Since Angular 13 does this as well?
https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296

Reply:
It's a feature toggle, so we might still need to do this.
AFAIK we can't check if the toggle is enabled or disabled.